packages/flutter/test/material/text_field_test.dart

```diff
@@ -2360,7 +2360,7 @@ void main() {
         break;
     }
   },
-    variant: TargetPlatformVariant.all(),
+    variant: TargetPlatformVariant.all(excluding: <TargetPlatform>{ TargetPlatform.iOS, TargetPlatform.macOS }),
```

Comment:
Nit: Change the title to say "on Apple platforms"?

Reply:
You're right. This also reminds me that I should add a test for Apple/non-Apple platforms.